README.md

```diff
@@ -11,8 +11,7 @@ We have provided three JSON files in `/data` that serve as source data (imagine
 ### `/cards`
 
 This endpoint returns a list of cards.
-
-- `backgroundImageUrl` should be the front-cover image found on the corresponding page template.
+- `imageUrl` should be the front-cover image found on the corresponding page template.
```

Comment:
I think this should be something like "Use the `imageUrl` of the first page template in the array" - I don't think we need to add an ID for `pages`